include/rofl/ofdpa/rofl_ofdpa_fm_driver.hpp

```diff
@@ -42,6 +42,11 @@ class rofl_ofdpa_fm_driver final {
   uint32_t disable_group_l2_unfiltered_interface(rofl::crofdpt &dpt,
                                                  uint32_t port_no);
 
+  uint32_t enable_group_l2_rewrite(
+      rofl::crofdpt &dpt, uint16_t id, uint32_t &interface, uint16_t vid = 0,
```

Comment:
nitpicking a bit, you don't need a reference for the interface, and please rename to port_no